src/meson.build

```diff
@@ -58,5 +54,24 @@ configure_file(
   install_dir: get_option('bindir')
 )
 
-install_data('workbench-api.vala', install_dir: datadir)
+install_data('workbench-api.vala', install_dir: pkgdatadir)
```

Comment:
@lw64 FYI

Reply:
the command to run `valac` needs to be adjusted, if they are different paths.

Reply:
here: https://github.com/sonnyp/Workbench/blob/main/src/Compiler.js#L16